src/index.css

```diff
@@ -0,0 +1,67 @@
+/* Reset default styles */
+* {
```

Comment:
I know that directly using elements here to style is bad practice, I have focused more on js than css